x-pack/filebeat/module/cisco/asa/test/asa.log-expected.json

```diff
@@ -3,15 +3,19 @@
         "@timestamp": "2018-10-10T12:34:56.000Z",
         "cisco.asa.message_id": "305011",
         "event.action": "firewall-rule",
+        "event.code": "305011",
         "event.dataset": "cisco.asa",
         "event.module": "cisco",
+        "event.original": "%ASA-6-305011: Built dynamic TCP translation from inside:172.31.98.44/1772 to outside:100.66.98.44/8256",
```

Comment:
I decided against keeping the Syslog header because in this case it only adds noise. I think we need to discuss this further, see also my comment above about keeping `log.original` for now.